bin/data-symlinks

```diff
@@ -18,15 +18,22 @@ for file in "$data"/*/===; do
     perl -pe '$_=lc; s/[^\w]+/-/g; s/^-//; s/-$//'
   )
 
+  baseid=$id
+  if [[ $(basename $id) != $id ]]; then
+    # we have a multitest, e.g. AB34/00
+    baseid=$(dirname $id)
+    mkdir -p "$data/meta/$baseid"
+  fi
   ln -s "$label" "$data/meta/$id.label"
-  ln -s "../$id" "$data/name/$slug"
+  [[ ! -e "$data/name/$slug" ]] && ln -s "../$baseid" "$data/name/$slug"
```

Comment:
Quotes are not needed for a `-e` in `[[`.

One useful way to learn all the places where quotes are and aren't needed is to remove them all, and only put them back where `shellcheck` says to.

Also you can split lines after things like `&&` and `|` without needing `\`.
And `! ... &&` better as `||`.
```
[[ -e "$data/name/$slug" ]] ||
  ln -s "../$baseid" "$data/name/$slug"
```